float32 forwardIRDist
float32 backwardIRDist
